pkg/kubelet/container/runtime.go

```diff
@@ -60,7 +60,7 @@ type Runtime interface {
 	GetPodStatus(*api.Pod) (*api.PodStatus, error)
 	// PullImage pulls an image from the network to local storage using the supplied
 	// secrets if necessary.
-	PullImage(image ImageSpec, secrets []api.Secret) error
+	PullImage(image ImageSpec, pullSecrets []api.Secret) error
```

Comment:
Not really as we have the implementations today. We had it and left it around in case we needed it in the future. I'm fine removing if that makes things simpler.